api/src/pcapi/core/educational/models.py

```diff
@@ -272,6 +272,12 @@ class CollectiveOffer(
 
     isActive: bool = sa.Column(sa.Boolean, nullable=False, server_default=sa.sql.expression.true(), default=True)
 
+    authorId = sa.Column(sa.BigInteger, sa.ForeignKey("user.id"), nullable=True)
+
+    author: sa_orm.Mapped["User"] | None = relationship(
+        "User", backref="collective_offers", foreign_keys=[authorId], uselist=False
```

Comment:
À quoi sert la backref ? A-t-on vraiment une raison de pouvoir accéder par l'ORM aux offres collectives créées par un utilisateur donné ? Si on n'en a pas besoin : 
```suggestion
        "User", foreign_keys=[authorId], uselist=False
```